src/main/java/no/nav/familie/integrasjoner/oppgave/OppgaveController.kt

```diff
@@ -52,8 +52,8 @@ class OppgaveController(private val oppgaveService: OppgaveService) {
         return success(oppgaveService.finnOppgaver(finnOppgaveRequest))
     }
 
-    @PostMapping(path = ["/mappe/sok"])
-    fun finnMapperV1(@RequestBody finnMappeRequest: FinnMappeRequest): Ressurs<FinnMappeResponseDto> {
+    @GetMapping(path = ["/mappe/sok"])
+    fun finnMapperV1(finnMappeRequest: FinnMappeRequest): Ressurs<FinnMappeResponseDto> {
```

Comment:
Dette er breaking change for mottak, som kan være greit ettersom det bare er å rekjøre tasks
Tenker det burde være en POST ettersom man sender med body?

Reply:
Trodde ikke den var tatt i bruk enda. En post skal i følghe rest gjøre oppdateringer. Vi har gjort noen unntak for spørringer som innnholder poteniselt sensitiv informasjon siden url og dermed query params logges. En get kan ikke ta i mot en requestBody, men den kan fint ta i mot en kotlin/java-objekt med request-feltene, slik som her.